index.html

```diff
@@ -0,0 +1,172 @@
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="">
+  <link rel="stylesheet" href="./css/reset.css">
+  <link rel="stylesheet" href="./css/main.css">
+  <link
+    href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,300&amp;display=swap"
+    rel="stylesheet">
+  <title>NAMM's 2020 Home Page</title>
+</head>
+
+<body>
+  <header id="header">
+    <nav class="small-nav bg-dark color-white-text">
+      <div class="fbt">
+        <a href="https://facebook.com">
+          <i class="fab fa-facebook-f" aria-hidden="true"></i>
+        </a>
+        <a href="http://twitter.com">
+          <i class="fab fa-twitter" aria-hidden="true"></i>
+        </a>
+      </div>
+      <ul class="small-bar-actions">
+        <li><button class="color-white-text">Logout</button></li>
+        <li><button class="color-white-text">Mypage</button></li>
+        <li><button class="color-white-text">English</button></li>
+      </ul>
+    </nav>
+
+    <button type="button" class="toggle-btn color-dark-text">
+      <img class="bars-fw1 bars-fw1-fs" src="./images/iconmonstr-menu-1.svg" alt="harmburger" aria-hidden="true">
+    </button>
+
+    <nav class="lg-navbar bg-white">
+      <a class="namm-logoo-a" href="./"><img class="namm-logoo" src="./images/download.png" alt="namm logo"></a>
+
+      <ul class="lg-nav-link">
+        <li><a class="nav-links" href="./about.html">About</a></li>
+        <li><a class="nav-links" href="#program">Program</a></li>
+        <li><a class="nav-links" href="#join">Join</a></li>
+        <li><a class="nav-links" href="#partners">Sponsor</a></li>
+        <li><a class="nav-links" href="#speakers">Guest</a></li>
+        <li><a class="nav-links join-namm-btn" href="#join-namm">NAMM 2022</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <section id="namm-2020-news" class="headline">
+      <div class="headline-content">
+        <h6 class="mission color-primary">"Promoting the pleasures and benefits of making music"</h6>
+        <div class="main-header">
+          <h1 class="header-1 ff-2">THE NATIONAL ASSOCIATION OF MUSIC MERCHANTS SHOW 2022</h1>
+        </div>
+        <div class="desc-box bg">
+          <p class="color-dark-text">we urge you to make plans to exhibit at the upcoming NAMM show, an in-person event
+            set to reunite our
+            global
+            communities, continue to power our industry's recovery, and create a more musical world.</p>
+        </div>
+        <div class="control-header-1">
+          <h3 class="header-2 color-dark-text ff-2">2022.20.01(THUR) ~ 23 (SUN)</h3>
+        </div>
+        <p class="color-dark-text">Anaheim Convention Center • Southern California <br><span
+            class="time color-dark-text ff-2"> - 10-AM PROMPT</span></p>
+      </div>
+    </section>
```

Comment:
This section is having too much white space

- [ ] Kindly reduce the space here
_See reference_
![image](https://user-images.githubusercontent.com/67957533/132857249-22b39877-1440-4ed7-986c-263c1433124e.png)
![image](https://user-images.githubusercontent.com/67957533/132857605-4d2f0c72-d626-458b-9c79-3e36f0328f09.png)